lib/network.sh

```diff
@@ -13,7 +13,7 @@ function network_address() {
   network=$2
   record=$3
 
-  result=$(python -c "import ipaddress; import itertools; print(next(itertools.islice(ipaddress.ip_network(u\"$network\").hosts(), $record - 1, None)))")
+  result=$(python3 -c "import ipaddress; import itertools; print(next(itertools.islice(ipaddress.ip_network(u\"$network\").hosts(), $record - 1, None)))")
```

Comment:
We already set alternatives to python3 on CentOS/RHEL8 - could we perhaps add that for Ubuntu to avoid the need for changing these?

https://github.com/metal3-io/metal3-dev-env/blob/master/centos_install_requirements.sh#L38

Reply:
Note #223 adds CentOS 8 support which we'll need to move to only python3

Reply:
> We already set alternatives to python3 on CentOS/RHEL8 - could we perhaps add that for Ubuntu to avoid the need for changing these?
> 
> https://github.com/metal3-io/metal3-dev-env/blob/master/centos_install_requirements.sh#L38

Yes, absolutely